apps/frontend-2/.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 18.8.0
```

Comment:
Denne fila fikk jeg commited ved et uhell. Kan fjerne den igjen.

Men ellers er jeg veldig glad i å bruke asdf for å sikre at alle har riktige versjoner av node etc som brukes i et prosjekt. Det fungerer slik at man definer en slik fil i prosjektet. Så settes versjoner automatisk når man kjører noe fra det repoet